From c6610c087ca95fcc03a5cbc64713134d15010747 Mon Sep 17 00:00:00 2001
From: Osama Khalid <osama@codinghazard.com>
Date: Sat, 22 Jun 2024 14:51:25 -0400
Subject: [PATCH] force data directory path and do not bleed build paths

---
 wsutil/path_config.h.in | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/wsutil/path_config.h.in b/wsutil/path_config.h.in
index 6539a5ab58..0318bc0056 100644
--- a/wsutil/path_config.h.in
+++ b/wsutil/path_config.h.in
@@ -1,10 +1,10 @@
 #ifndef __PATH_CONFIG_H__
 #define __PATH_CONFIG_H__
 
-#define INSTALL_PREFIX  "@PATH_INSTALL_PREFIX@"
-#define DATA_DIR        "@PATH_DATA_DIR@"
-#define DOC_DIR         "@PATH_DOC_DIR@"
-#define PLUGIN_DIR      "@PATH_PLUGIN_DIR@"
-#define EXTCAP_DIR      "@PATH_EXTCAP_DIR@"
+#define INSTALL_PREFIX  "/"
+#define DATA_DIR        "/wireshark"
+#define DOC_DIR         "/docs"
+#define PLUGIN_DIR      "/plugins"
+#define EXTCAP_DIR      "/extcap"
 
 #endif
-- 
2.43.0

