From 56160a1aeabb9513c2e93cd49c40f0c56514c58d Mon Sep 17 00:00:00 2001
From: Devan Lai <devan.lai@gmail.com>
Date: Sun, 5 Nov 2023 10:31:35 -0800
Subject: [PATCH] Export lrexlib

---
 epan/wslua/lrexlib/CMakeLists.txt | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/epan/wslua/lrexlib/CMakeLists.txt b/epan/wslua/lrexlib/CMakeLists.txt
index 1f5f13f278..a60b3f916f 100644
--- a/epan/wslua/lrexlib/CMakeLists.txt
+++ b/epan/wslua/lrexlib/CMakeLists.txt
@@ -24,3 +24,17 @@ add_compile_definitions(
 	VERSION=\"2.9.1\"
 	PCRE2_CODE_UNIT_WIDTH=8
 )
+
+set_target_properties(lrexlib PROPERTIES
+	PREFIX "lib"
+	LINK_FLAGS "${WS_LINK_FLAGS}"
+	FOLDER "DLLs"
+	INSTALL_RPATH "${LIBRARY_INSTALL_RPATH}"
+)
+
+install(TARGETS lrexlib
+	EXPORT WiresharkTargets
+	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+	LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+	ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+)
-- 
2.34.1

